Firmware/USMArduinoFirmware/USMArduinoFirmware.ino

```diff
@@ -143,11 +149,19 @@ Adafruit_MCP23017 mcp23017[MCP_MAX_COUNT];
 // Input handlers
 USM_Input usmInput[MCP_MAX_COUNT];
 
-// Ethernet client
-EthernetClient ethernet;
-
-// MQTT client
-PubSubClient mqtt_client(mqtt_broker, mqtt_port, mqttCallback, ethernet);
+#if defined (ESP8266) || defined (ESP32)
```

Comment:
@moinmoin-sh I am not an expert in these different MCUs, but I know @jonoxer is talking about building this URC with an ESP32 and ethernet/POE - so wouldn't this change mean the firmware would not work for that config? Since it assumes if you are using an ESP32, you are using WiFi?

Reply:
Yes, this assumes ESP32 is WiFi. I haven't ever seen an ESP32 with ethernet - so I don't know how that needs to be configured.

Reply:
I think in the case of the new URC, Jon will be adding the ethernet himself, but my point is the firmware wouldn't work with this change. Or it would work, but would ignore ethernet.